### DISCUSSION TITLE
How can I access InfluxDB from my network?

### DISCUSSION THREAD

**Discussion by blissi** [hassio-addons/addon-influxdb#299] [General]
Hallo,
my Raspberry PI with HassOS has the IP address 192.168.0.2 and Home Assistant is reachable under homeassistant.local in my network.
Unfortunately for some reason I can only connect to the InfluxDB add-on on Home Assistant itself. When I try this on my local computer, I always get a 404.

I have a small Python script for testing access:

```
with InfluxDBClient(url="http://homeassistant.local:8086", username="...", password="...", org="home") as client:
    query_api = client.query_api()
    tables = query_api.query('from(bucket:"home") |> range(start: -10m)')
    for table in tables:
        print(table)
```

I have also tried with url="http://192.168.0.2:8086" but that also didn't work. And in the addon-on configuration I have disabled SSL.

What is wrong here?

Thanks,
Blissi

**Comment by kalgecin** [comment]
I too am looking for a way to push data into influxdb from a script running on local network. I cannot login into influxdb cos /api/v2/signin returns 404 
```
INFO: --> POST http://192.168.12.225:8086/api/v2/signin
INFO: <-- 404 Not Found http://192.168.12.225:8086/api/v2/signin (10ms)
```
